src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx

```diff
@@ -17,25 +17,31 @@ export const IsomerDetailGroupView = ({
   const selected = activePos >= startPos && activePos <= endPos
 
   const { onDrawerOpen } = useEditorDrawerContext()
+
+  // todo: get drawer reviewed in staging
+  const isProd = process.env.REACT_APP_ENV === "production"
+
   const otherButtons = (
     <>
       <Box
-        display={selected ? "block" : "none"}
+        display={selected ? "flex" : "none"}
         position="absolute"
         width="100%"
+        justifyContent="end"
       >
         <HStack
+          bgColor="interaction.mainLight.default"
+          borderRadius="0.25rem"
+          border="1px solid"
           borderColor="base.divider.medium"
-          display="flex"
-          justifyContent="flex-end"
+          boxShadow="0px 8px 12px 0px rgba(187, 187, 187, 0.50)"
+          px="0.5rem"
+          py="0.25rem"
+          mr="-0.5rem"
+          mt="-1.5rem"
         >
-          <Box
-            backgroundColor={selected ? "white" : "transparent"}
-            marginTop="-0.25rem"
-            boxShadow="0px 0px 4px 0px rgba(0, 0, 0, 0.8)"
-            borderRadius="4px"
-          >
-            <Tooltip label="Edit accordion grid" hasArrow placement="top">
+          {!isProd && (
+            <Tooltip label="Edit accordion block" hasArrow placement="top">
```

Comment:
it feels very weird to see this here - can we cut this out and raise another PR? this seems like testing code going into production

Reply:
the code is already in main line hahah, just so happens hidden behind feature flag. 
this means the removal of a fair bit of code + then re-reviewal of already reviewed code, which feels counterproductive, thoughts?